runtime/jcl/common/jclcinit.c

```diff
@@ -280,6 +280,7 @@ jint initializeKnownClasses(J9JavaVM* vm, U_32 runtimeFlags)
 					Trc_JCL_initializeKnownClasses_ClassRefNotResolvedForInstanceFieldRef(vm->mainThread, romFieldConstantPool[i].classRefCPIndex, i);
```

Comment:
Do we also need an assert here?

Reply:
That will be a behaviour change though, it doesn't return `JNI_ERR`.

Reply:
We should add something in initializeRequiredClasses().
https://github.com/eclipse-openj9/openj9/blob/master/runtime/jcl/common/jclcinit.c#L746

Isn't there a place that iterates classes in the constant pool?

Reply:
Seems it would be beneficial to have tracepoints in all the places that can fail in initializeRequiredClasses() and other places. I guess failures happen so rarely.

Reply:
iirc, last time VM exit silently probably was couple of years ago, I think these assertions are sufficient to provide enough hint about what has to be fixed.

Reply:
Ya, I didn't expect you to fix everything in this PR, just commenting. I still think it's beneficial to at least add another tracepoint and assert at https://github.com/eclipse-openj9/openj9/blob/master/runtime/jcl/common/jclcinit.c#L746 as there are a couple of version specific classes in there.

Reply:
Added.